src/scss/_faq.scss

```diff
@@ -0,0 +1,123 @@
+//---------------Break Points---------------//
+//25em (400px)
+//45em (720px)
+//50em (800px)
+//61.25em (980px)
+//87.5em (1400px)
+
+.faq {
+  background-color: $color-orange;
+  text-align: center;
+  padding: 4em 0; //64px 0
+}
+
+.faq__heading {
+  width: 10em; //160px
+  font-family: $font-egyptian;
+  font-size: 1em; //16px
+  color: $color-light-gray;
+  border-top: 2px solid $color-light-gray;
+  border-bottom: 2px solid $color-light-gray;
+  padding: 0.25em 0; //4px 0
+  margin: auto;
+
+  @media(min-width: 61.25em) {
+    font-size: 1.5em; //24px
+  }
+}
+
+.faq__list__links {
+  width: 100%;
+  margin: 4em auto; //64px 0
+}
+
+.faq__link {
+  width: 48%;
+  display: inline;
+  font-family: $font-egyptian;
+  font-size: 2.25em; //36px
+  color: $color-tan;
+  text-decoration: none;
+  padding: 0.5em; //8px
+  border: 2px solid $color-brown;
+
+  &:hover {
+    background-color: $color-brown;
+  }
+  
+  &:active {
```

Comment:
Could you consolidate this with the selector above like so:

`&:hover, &:active {`

Might tighten up the code a bit.